go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -294,3 +305,17 @@ func runOvnKube(ctx *cli.Context) error {
 	wg.Wait()
 	return nil
 }
+
+// libovsdbClientOnConnect handles specific actions taken against libovsdbClient
+// on first connection and further reconnections.
+func libovsdbClientOnConnect(client libovsdbClient.Client) error {
+	// add event handlers as needed
+	// client.Cache().AddEventHandler(...)
+
+	// Monitor all, but this can be refined to specific tables
+	err := client.MonitorAll(nil)
```

Comment:
I think @trozet mentioned that we couldn't monitor sbdb because it broke everything... which could be down to either client-side handling of update notifications, or server-side not having enough cpu time to process changes, monitors and raft. I don't think it's safe to just `MonitorAll` all the things.

Once https://github.com/ovn-org/libovsdb/pull/176 merges (and we bump the dependency) this function will become obsolete and the monitor/handler setup will need to move to `util.NewNBClient` or `util.NewSBClient`. I think it may make sense to move it inline in those functions now.
e.g `onConnectFn := func () ...` which would also allow you to setup `MonitorAll` for NBDB by default on no monitor for SBDB.